yarn.lock

```diff
@@ -2198,6 +2198,11 @@ react-dom@^18:
     loose-envify "^1.1.0"
     scheduler "^0.23.2"
 
+react-intersection-observer@^9.13.0:
```

Comment:
How can you find the perfect package or library every time? Respectful.

Reply:
I learn a lot through YouTube and Velog Trending. There's a lot of good information [here](https://kofearticle.substack.com/) too!